karma-shared.conf.js

```diff
@@ -10,7 +10,7 @@ module.exports = function(config, specificOptions) {
     browserDisconnectTimeout: 10000,
     browserDisconnectTolerance: 2,
     browserNoActivityTimeout: 30000,
-    reporters: ['spec'],
+    reporters: ['dots'],
```

Comment:
Yes, because it doesn't generate a dot for each test, so the test results are easier to scroll through - I changed that a while ago, but accidentially added it locally too. Locally, dots is better because you often follow the progress while it's testing